### ISSUE TITLE
is it useful, I tested ringloss, however it is not useful for face recognition 

### ISSUE THREAD

**Issue by IrvingShu** [Paralysis/ringloss#5]

**Comment by sachinpuranik99** [comment]
The original paper claims that the RingLoss along with Softmax or Sphereface loss is able to surpass state-of-the-art accuracy in face verification/recognition. Maybe you can check for bugs in your implementation/evaluation. If you are sure about your implementation, please get in touch with the original paper authors.

**Comment by Paralysis** [comment]
@IrvingShu As @sachinpuranik99 noted, face recognition is much messier than MNIST. It depends on lots of your factors, e.g. data processing, training hyper-parameters, initializations. I suggest you follow the author's implementation detail and check if you have any bugs with them. **Also, try initializing RingLoss training with an intermediate Softmax/Sphereface model. It will converge faster.**
